Dockerfile-ubuntu1804

```diff
@@ -25,9 +25,10 @@ RUN ln -s /usr/bin/clang-9 /usr/bin/clang \
 
 # Install Python 3.8
 RUN add-apt-repository -y ppa:deadsnakes/ppa \
-  && apt update && apt -y install python3.8 python3.8-venv python3.8-dev \
-  && python3.8 -m ensurepip && python3.8 -m pip install --upgrade pip setuptools wheel \
-  && python3.8 -m pip install pytest
+  && apt-get update && apt-get -y install python3.8 python3.8-venv python3.8-dev \
```

Comment:
why is `apt-get` needed and not just `apt`?

Reply:
Just to avoid warning about non stable interface